src/code/fv-components/gem-set.js

```diff
@@ -0,0 +1,63 @@
+import React, {PropTypes} from 'react';
+import { getChallengeScores } from '../reducers/helpers/challenge-progress';
+
+export const GemView = ({challengeNum, score, highlight}) => {
```

Comment:
I followed the pattern of `BasketSetView` here, which adds `BasketView` as an internal component